4_model_evaluation.yml

```diff
@@ -0,0 +1,55 @@
+target_default: 4_model_evaluation
+
+packages:
+  - dplyr
+  - scipiper
+
+sources:
+  - code/Metric_fetch.R
+  - code/Metrics_process.R
+  - code/Metric_visualize.R
+  - code/Metric_calculate.R
+
+targets:
+  4_model_evaluation:
+    depends:
+     - "4_model_evaluation/out/gen_metrics_all_obs.csv"
+     - "4_model_evaluation/out/gen_metrics_by_seg.csv"
+     - "4_model_evaluation/out/gen_metrics_by_year.csv"
+     - "4_model_evaluation/out/gen_metrics_by_month.csv"
+     - "4_model_evaluation/out/exceedance_metrics_all_obs.csv"
+     - "4_model_evaluation/out/exceedance_metrics_by_seg.csv"
+     - "4_model_evaluation/out/exceedance_metrics_by_year.csv"
+     - "4_model_evaluation/out/exceedance_metrics_by_month.csv"
+     - "4_model_evaluation/out/max_metrics_by_seg_year.csv"
+     
+
```

Comment:
For now, can you create a target that is long_dat? It would look something like:
```
long_dat:
     command: readRDS('data/name_of_long_data.rds')

```

Or, if there's processing you do on long dat, you could create a function that does everything you need it to (read it in, process, etc).